### DISCUSSION TITLE
Zigbee2mqtt service won't never start but manually always

### DISCUSSION THREAD

**Discussion by henkiejan1** [Koenkk/zigbee2mqtt#11914] [General]
Hi everyone,

I got this problem for a long time but can't get it fixed. I run Zigbee2MQTT on Armbian Focal 20,.11 and mostly, so not all, the service wont start. The service looks like in a reboot loop. Start time resets. There is nothing to read in the log file.

But if i run manually in /opt/zigbee2mqtt with the command sudo npm start it runs always. The same user account is in the configuration file. Also tried the suggestions here: https://www.zigbee2mqtt.io/information/zigbee2mqtt_fails_to_start.html But again: it runs without any problems when i run zigbee2mqtt manually.

So please help!

My service:
`[Unit]
Description=zigbee2mqtt
After=network.target

[Service]
ExecStart=/usr/bin/npm start
WorkingDirectory=/opt/zigbee2mqtt
StandardOutput=inherit
# Or use StandardOutput=null if you don't want Zigbee2MQTT messages filling syslog, for more options see systemd.exec(5)
StandardError=inherit
Restart=always
User=hj

[Install]
WantedBy=multi-user.target
`

**Comment by loves80z** [comment]
I have the same issue. So far I found that if I check the node version under my user it is a different version than the root user. Not sure what user the service is run under, but if there is something different about the other user perhaps it is preventing running as a service
